live-editing/services/DependencyResolver.ts

```diff
@@ -88,6 +110,10 @@ export class DependencyResolver {
         SHARED_DEPENDENCIES.forEach((d) => this._chartsDependencies.add(d));
         CHARTS_DEPENDENCIES.forEach((d) => this._chartsDependencies.add(d));
 
+        this._mapsDependencies = new Set<string>();
+        SHARED_DEPENDENCIES.forEach((d) => this._chartsDependencies.add(d));
```

Comment:
This has to be _mapDependencies instead of _chartsDependencies

Reply:
I have corrected it